src/ContactDetails/ContactDetails.js

```diff
@@ -0,0 +1,31 @@
+import './ContactDetails.scss';
+import React, { useState } from 'react';
+import { useQuery } from '@apollo/client';
+
+function ContactDetails({query, id, resName, type}) {
```

Comment:
I'm glad you broke this out into its own reusable component. Makes a lot of sense.